include/json/writer.h

```diff
@@ -28,29 +28,29 @@ class Value;
 
 /**
 
-Usage:
-\code
-  using namespace Json;
-  void writeToStdout(StreamWriter::Factory const& factory, Value const& value) {
-    std::unique_ptr<StreamWriter> const writer(
-      factory.newStreamWriter());
-    writer->write(value, &std::cout);
-    std::cout << std::endl;  // add lf and flush
-  }
-\endcode
+* Usage:
+*  \code
+*  using namespace Json;
+*  void writeToStdout(StreamWriter::Factory const& factory, Value const& value) {
+*    std::unique_ptr<Json::StreamWriter> const writer(
```

Comment:
"using namespace Json" above makes this unnecessary.

Reply:
> "using namespace Json" above makes this unnecessary.

Oops, I didn't see it.